src/App.js

```diff
@@ -1,26 +1,49 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import Api from './Api';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: []
+    };
+  }
+
+  componentDidMount() {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+    .then(response => response.json())
+    .then(data => this.setState({ movies: data.movies }));
```

Comment:
In the next step, we are looking at adding error handling. 
We are still not sure if we want to split this into another file later